### ISSUE TITLE
Docs don't include all available loss functions

### ISSUE THREAD

**Issue by rafaqz** [JuliaML/LossFunctions.jl#111]
So the non-included funcitons are a little hard to find. CrossentropyLoss and PoissonLoss are examples.

**Comment by xiaodaigh** [comment]
Wow. Didn't even know they existed!
